docs/how-to-write-a-contract/built-ins.md

```diff
@@ -110,7 +114,11 @@ const s2 = toByteString('hello', true) // '68656c6c6f', 5 bytes
 len(s2) // 5
 ```
 
-- `reverseByteString(b: ByteString, size: number): ByteString` Returns reversed bytes of `b` which is of `size` bytes. Note `size` must be a [compiled-time constant](./how-to-write-a-contract.md#compile-time-constant). It is often useful when converting a number between little-endian and big-endian.
+- `reverseByteString(b: ByteString, size: number): ByteString` Returns reversed bytes of `b` which is of `size` bytes.
+
+:::note
+`size` must be a [compiled-time constant](./how-to-write-a-contract.md#compile-time-constant). It is often useful when converting a number between little-endian and big-endian.
+:::
```

Comment:
`reverseByteString(b: ByteString, size: number): ByteString` Returns reversed bytes of `b` which is of `size` bytes. It is often useful when converting a number between little-endian and big-endian.

:::note
`size` must be a [compiled-time constant](./how-to-write-a-contract.md#compile-time-constant).
:::